### ISSUE TITLE
Create Index Link Does Not Work

### ISSUE THREAD

**Issue by kimfucious** [firebase/firebase-js-sdk#7578]
### Operating System

macOS 13.4.1 (c) (22F770820d)

### Browser Version

Chrome  115.0.5790.170 (Official Build) (arm64)

### Firebase SDK Version

9.21.0

### Firebase SDK Product:

Analytics, Auth, Database, Firestore, Functions, Storage

### Describe your project's tooling

NextJS v13.0.2 with React v18.2.0 under the hood. 

### Describe the problem

When a compound query is made with something like this:
```js
let query = productsRef
    .orderBy("isFeatured", "desc")
    .orderBy("updatedAt", "desc")
    .limit(options.size);
```
An error is shown in the console:
```console
'The query requires an index. You can create it here: https://console.firebase.google.com/v1/r/project/<project-name>/firestore/indexes?create_composite=<big-long-string>'
```
This is expected and documented [here](https://firebase.google.com/docs/firestore/query-data/indexing?hl=en&authuser=0&_gl=1*1e1c5j9*_ga*NzAzODU2NTgxLjE2Nzg3MDA1ODE.*_ga_CW55HF8NVT*MTY5MjcwODU2OC4yMjAuMS4xNjkyNzA4ODkyLjAuMC4w#create_a_missing_index_through_an_error_message).

When I visit that link in Chrome, I am presented with an option to choose a Google account (I have several).

When I select the appropriate Google account, the firebase project is opened to Cloud Firestore => Indexes.

No indexes are shown and no dialog is presented to create the index.

Here's a screenshot of blankness:
![image](https://github.com/firebase/firebase-js-sdk/assets/26792769/3f84bb70-7754-4e64-97b4-1fdf837e99b3)

Here's the thing:  I know that there are indexes there, and I have created some of these using links provided as described above.  The problem is that the links have stopped working.

I can create indexes manually, and when I access the Indexes tab in FireStore Database directly (not via a link), they are visible.

Here's a screen shot when accessing without using a link:
![image](https://github.com/firebase/firebase-js-sdk/assets/26792769/8f3d14b1-a67a-4b45-913a-7042ce02330a)

Note:  When the indexes tab is blank in the browser, the following errors are shown in the browser console:
```console
Failed to load resource: the server responded with a status of 404 ()
Failed to load resource: the server responded with a status of 404 ()
Failed to load resource: the server responded with a status of 404 ()
Failed to load resource: the server responded with a status of 404 ()
Error with Permissions-Policy header: Unrecognized feature: 'ch-ua-form-factor'.
```

### Steps and code to reproduce issue

1. Open missing index link in Chrome.
2. See page with no indexes and no create index dialog.

**Comment by cherylEnkidu** [comment]
Hi @kimfucious 

Thanks for reporting this issue. I cannot reproduce it on my side. Could you try log out in command line `firebase logout`, then `firebase login`, and run the query to see if the link works this time?

**Comment by kimfucious** [comment]
Hi @cherylEnkidu.

Thanks for the follow-up.

Unfortunately logging out and back in with the firebase CLI did not resolve the issue.

**Comment by christianfunginet** [comment]
Same problem. Follow post

**Comment by kimfucious** [comment]
@tom-andersen 

I'd file the bug, but this magically started working again (kinda).

To clarify, as this wasn't working I created needed indexes manually.

The other day, I created a query that threw an error telling me I needed an index.  Rather than create the index manually, I used the link in the error, and it worked.

I cannot say whether or not the problem is fixed permanently, but if it happens again (should I need another index), I'll let you folks know in a new ticket.

Thanks.
